ql/experimental/termstructures/basisswapratehelpers.hpp

```diff
@@ -53,6 +53,8 @@ namespace QuantLib {
 
         Real impliedQuote() const override;
         void accept(AcyclicVisitor&) override;
+        // NOLINTNEXTLINE(cppcoreguidelines-noexcept-swap,performance-noexcept-swap)
+        ext::shared_ptr<Swap> swap() const { return swap_; }
```

Comment:
most of ql's codes use naming `underlyingSwap` instead of `swap` though those are related to swaption-type classes.